examples/insecure_server.toml

```diff
@@ -11,5 +11,7 @@ tls_key = "/tmp/kanidm/key.pem"
 log_level = "verbose"
 
 domain = "idm.example.com"
+domain_display_name = "Kani's Crab Shack"
```

Comment:
Not sure about this being in the server.toml, since this is a "domain wide" setting, not a "per-server" setting, and it's not critical to have in place for bootstrap (like the domain name is).